specification/chaos/resource-manager/Microsoft.Chaos/stable/2023-11-01/experiments.json

```diff
@@ -0,0 +1,547 @@
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of executions returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experimentExecutionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all executions of an Experiment.": {
+            "$ref": "./examples/ListExperimentExecutions.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}": {
+      "get": {
+        "description": "Get an execution of an Experiment resource.",
+        "tags": [
+          "Experiments"
+        ],
+        "operationId": "Experiments_GetExecution",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/executionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Execution returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experimentExecution"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get the execution of a Experiment.": {
+            "$ref": "./examples/GetExperimentExecution.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}/details": {
+      "post": {
```

Comment:
>  "post": { [](http://example.com/codeflow?start=4&length=11)

Why this change from LIST to post? In general please provide justifications when you open the PR to avoid back and forths that result in delays.

Reply:
this is the one API we added in this version.
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions - LIST all executions
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId} - GET an execution
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/executions/{executionId}/details - returns the detailed log of an execution (hence this is an action on the child resource) This was feedback I received from you during office hour.

Reply:
ok, when you raise PRs pl provide context on whats being attempted. Its impossible to retain context from other discussions as we go thru roughly 50 - 70 PRs per week and many design discussions

Reply:
Also pl make sure the first commit is an exact copy of the previous API version and changes are added in subsequent commits. This is called out in the PR summary. A separate commit makes it easy to review otherwise the diffs with the first sommit wont show up correctly and the reviewer wouldnt know what the real changes are

Reply:
we have been seeing issue with this for a while now. I don't understand what is the ask here. From what I understand the first commit is an exact copy of the base API. We generated this PR with OpenAPI hub.

Reply:
ping me on teams, I will explain